drivers/docker/driver_unix_test.go

```diff
@@ -299,10 +299,10 @@ func TestDockerDriver_BindMountsHonorVolumesEnabledFlag(t *testing.T) {
 		},
 		{
 			name:            "relative local driver",
```

Comment:
~~ Maybe update the name to reflect this being treated as a volume name rather than a relative path.